drape/support_manager.cpp

```diff
@@ -93,6 +93,10 @@ void SupportManager::Init(ref_ptr<GraphicsContext> context)
 //    settings::Set(kSupportedAntialiasing, m_isAntialiasingEnabledByDefault);
 //  }
 
+#ifdef OMIM_OS_DESKTOP
```

Comment:
The `OMIM_OS_DESKTOP` is currently massively abused, because it is also used on Linux mobile devices, and the last plan is to use a [single convergent binary for desktop and mobile devices](https://github.com/organicmaps/organicmaps/pull/2947#issuecomment-1440845368).
So I think `OMIM_OS_DESKTOP` should be renamed to something `QT` in it.

And IMHO, this AA setting should either be unconditionally on/off for `QT` (regardless of mobile or desktop DPI), or somehow dynamically set. Perhaps based on [logicalDpiX](https://doc.qt.io/qt-5/qpaintdevice.html#logicalDpiX).

Reply:
Another thought is, why limiting this to `Qt`? Aren't there more than 60+ inch android touch screen devices with low DPI in many classroom and meeting rooms? I think they would be handy for showing maps for educational or planning purposes.

Reply:
OMIM_OS_DESKTOP means:
1. It is not Android and not iOS. 
2. It uses Qt.

For Mobile Linux, it also conflicts now with OMIM_OS_MOBILE.

Enabling it may reduce performance in some cases. And increase battery usage on mobile devices. Did anyone from mobile Linux report any related issues?

Did someone check why it was disabled?

@endim8 did you test if enabling it works?

Reply:
It was disabled by default for battery reasons: https://github.com/organicmaps/organicmaps/blob/03082c1eb2c065096883e5b2d7ebab51474a7e5b/drape/support_manager.cpp#L81-L94

I can confirm that it works.

The point about Linux mobile devices is good, maybe we can just refine OMIM_OS_DESKTOP to exclude them? A complete refactoring to remove OMIM_OS_* is probably more effort than reward, as AFAIK it's used quite a lot.

Reply:
> actually no, because mobile devices with low dpi are usually old and slow... I

So isn't the currently commented out code that would use the renderer name to determine AA support
has the same approach, to determine which devices are too "old and slow"?
In theory that could work on Linux and Android.

Reply:
There could also be issues with the driver on some mobiles (artifacts, crashes). I would definitely NOT change it for iOS and Android. And consider enabling it only on low-dpi desktops, where the battery isn't an issue.

Reply:
IMO the right way could be to disable AA on all mobiles (to save the battery), and enable it only on low-resolution desktops (to have faster rendering on high-resolution ones). So we need two dynamic checks:
1. Is the current Qt version running on a mobile device?
2. If not, then is the DPI low? (need to find the good threshold for "low").